hack/jenkins/e2e.sh

```diff
@@ -398,30 +398,31 @@ case ${JOB_NAME} in
     : ${PROJECT:="kubernetes-jenkins"}
     ;;
 
-  # Sets up the GCE soak cluster weekly using the latest 1.1 release.
-  kubernetes-soak-weekly-deploy-gce-release-1.1)
-    : ${E2E_CLUSTER_NAME:="gce-soak-weekly-release-1.1"}
+  # Sets up the GCE soak cluster weekly using the latest 1.1 ci release.
+  kubernetes-soak-weekly-deploy-gce-1.1)
+    : ${E2E_CLUSTER_NAME:="gce-soak-weekly-1.1"}
```

Comment:
You might consider using `gce-soak-weekly-1-1` (with `-` instead of `.`) for the cluster name, too.